ui/k-toolbar/main.kytos

```diff
@@ -1,343 +1,431 @@
 <template>
-  <k-toolbar-item icon="compass" tooltip="Napp Pathfinder">
-    <div class="scroll">
-      <k-accordion>
-        <k-accordion-item title="Best Paths">
-         <k-dropdown icon="circle-o" title="Source:" :options="get_interfaces"
-          :value.sync="source"></k-dropdown>
-         <k-dropdown icon="circle-o" title="Destination:" :options="get_interfaces"
-          :value.sync ="destination"></k-dropdown>
-         <k-select icon="link" title="Undesired links:" :options="get_links"
-          :value.sync ="undesired_links"></k-select>
-          <div class="metric">
-            <div class="checkbox">
-              <k-checkbox :model.sync = "checked_list" :value = "'bandwidth'"></k-checkbox>
+    <k-toolbar-item icon="compass" tooltip="Napp Pathfinder">
+      <div class="scroll">
+        <k-accordion>
+          <k-accordion-item title="Best Paths">
+  
+          <!-- this is where the search bars are -->
```

Comment:
No need to add the comments in the template html separating the components. In some cases they might be handy, but since this is a very small component no need to add this comment here, so if you could remove that'd be great.